src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -457,6 +457,14 @@ void CascadiaSettings::_ValidateMediaResources()
     {
         if (!profile.BackgroundImagePath().empty())
         {
+            // checks if the if the user would like to copy their desktop wallpaper

```

Comment:
Got it, will make those changes.

Reply:
@carlos-zamora Let me know what you think. I was a little bit confused where exactly the changes should be made. Sorry for the "slow down". I am still learning the code base and trying to understand the flow. Are there any diagrams made to help with this? The bright side is I'm learning a ton on every revision.

- [ ] - profile.BackgroundImagePath() is never changed anymore. So it stays as "DesktopWallpaper" all the way through. Not sure if this is exactly what you meant by it being a static string.
- [X] - logic is now down in profile.cpp and we set the profile.ExpandedBackgroundImagePath() right when we expand it.
- [X] - Given this method,  `_ValidateMediaResources` should accept "DesktopWallpaper", given it then uses `profile.ExpandedBackgroundImagePath()` for validation.
- [X] - Added a test to `DeserializationTests.cpp`, but I do not know how to run tests, so I wasnt able to test it.... (some guidance on this would be appreciated).

I also fixed the comment error.